indigo_app/js/components/DocumentTOCView.vue

```diff
@@ -113,6 +113,14 @@ export default {
         return headingText;
       };
 
+      const getDefinitionTitle = (node) => {
+        const term = node.getAttribute('refersTo');
+        const result = this.model.xpath(`.//a:def[@refersTo="${term}"]`, node);
+        if (result.snapshotLength > 0) {
+          return result.snapshotItem(0).textContent;
+        }
```

Comment:
I agree that have it there is useful, so perhaps use something like `(term missing)` if there is no text just to handle that edge case and ensure the user has something clickable?